python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/abstract_interp_lib_gen.py

```diff
@@ -0,0 +1,1080 @@
+
+def aten〇rsub〇Scalar〡shape(self: List[int], other: float, alpha: float = 1) -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+def aten〇to〇dtype〡shape(self: List[int], dtype: int, non_blocking: bool = False, copy: bool = False, memory_format: Optional[int] = None) -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+def aten〇to〇dtype_layout〡shape(self: List[int], dtype: Optional[int] = None, layout: Optional[int] = None, device: Optional[device] = None, pin_memory: Optional[bool] = None, non_blocking: bool = False, copy: bool = False, memory_format: Optional[int] = None) -> List[int]:
+    return self
+
+def aten〇to〇device〡shape(self: List[int], device: device, dtype: int, non_blocking: bool = False, copy: bool = False, memory_format: Optional[int] = None) -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+def aten〇to〇other〡shape(self: List[int], other: List[int], non_blocking: bool = False, copy: bool = False, memory_format: Optional[int] = None) -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+def aten〇type_as〡shape(self: List[int], other: List[int]) -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+def aten〇dropout〡shape(input: List[int], p: float, train: bool) -> List[int]:
+    return upstream_shape_functions.unary(input)
+
+def aten〇gelu〡shape(self: List[int], approximate: str = "none") -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+def aten〇contiguous〡shape(self: List[int], memory_format: int = 0) -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+def aten〇clone〡shape(self: List[int], memory_format: Optional[int] = None) -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+def aten〇lift_fresh_copy〡shape(self: List[int]) -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+def aten〇_log_softmax_backward_data〡shape(grad_output: List[int], output: List[int], dim: int, input_dtype: int) -> List[int]:
+    return upstream_shape_functions.unary(grad_output)
+
+def aten〇eq〇Scalar〡shape(self: List[int], other: float) -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+def aten〇ne〇Scalar〡shape(self: List[int], other: float) -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+def aten〇gt〇Scalar〡shape(self: List[int], other: float) -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+def aten〇ge〇Scalar〡shape(self: List[int], other: float) -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+def aten〇le〇Scalar〡shape(self: List[int], other: float) -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+def aten〇lt〇Scalar〡shape(self: List[int], other: float) -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+def aten〇add〇Scalar〡shape(self: List[int], other: float, alpha: float = 1) -> List[int]:
+    return upstream_shape_functions.unary(self)
+
+@check_dtype_function([
+    Invocation(TensorOfShape(1, 1, 1, dtype=torch.float32), other=0),
```

Comment:
Is there a reason you're using 1,1,1 instead of just 1 or 1,1 or []?

You may want to define helpers NonZeroDTensorWithDtype(torch.int64) and ZeroDTensorWithDtype(torch.int64) to make the testing intent for the dtype functions clearer. (that is, not even have a place to write the shape, since it is immaterial to the test)